lib/config/rpcProviderProdConfig.json

```diff
@@ -3,6 +3,12 @@
     "chainId": 43114,
     "useMultiProviderProb": 1,
     "providerInitialWeights": [9, 1, 0],
-    "providerUrls": ["WEB3_RPC_43114", "WEB3_RPC_43114_QUICKNODE", "WEB3_RPC_43114_NIRVANA"]
+    "providerUrls": ["WEB3_RPC_43114_INFURA", "WEB3_RPC_43114_QUICKNODE", "WEB3_RPC_43114_NIRVANA"]
+  },
+  {
+    "chainId": 10,
+    "useMultiProviderProb": 1,
+    "providerInitialWeights": [1, 1, 1, 1],
```

Comment:
you want to equally distribute between them? Or do we want to do 100% infura and just run the rest in shadow? i thought the latter

Reply:
Nice catch. Fixed as suggested.